tests/notebooks_test.py

```diff
@@ -3,16 +3,20 @@
 import nbformat
 from nbconvert.preprocessors.execute import CellExecutionError
 from nbconvert.preprocessors import ExecutePreprocessor
+import pytest
 
 from allennlp.common.testing import AllenNlpTestCase
 
 class TestNotebooks(AllenNlpTestCase):
+    @pytest.mark.skip()
```

Comment:
If you're disabling these, can you open an issue to fix them before the next release?

Reply:
oops, I only disabled them because I didn't want to iterate on the notebooks until we'd agreed what the new API should look like. I fixed the notebooks and re-enabled these tests.